addons/mail/static/tests/discuss/core/web/discuss.test.js

```diff
@@ -29,21 +29,24 @@ test("can create a new channel", async () => {
     const pyEnv = await startServer();
     onRpcBefore((route, args) => {
         if (
-            route.startsWith("/mail") ||
             route.startsWith("/discuss/channel/create_channel") ||
             route.startsWith("/discuss/channel/messages") ||
             route.startsWith("/discuss/search")
         ) {
             asyncStep(`${route} - ${JSON.stringify(args)}`);
         }
+        if (
+            (route === "/mail/data" && args.fetch_params.includes("channels_as_member")) ||
+            route === "/mail/inbox/messages"
+        ) {
+            asyncStep(`${route} - ${JSON.stringify(args)}`);
+        }
```

Comment:
Nitpick: could be with the other `if` conditions above, to not repeat same LOC `asyncStep()`